scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -310,6 +349,50 @@ def _get_containers(client, ignore_container=None, restrict_to_container=None, l
 
     return result
 
+def get_attributes_and_config_from_labels( labels, docker_options ):
+    """
+        Takes a dict of labels and splits it in to two separate attributes and config dicts.
+
+        @param: labels - a dict containing all the labels
```

Comment:
I would say `a dict containing all the Docker labels for a container`

Reply:
Done